lib/matplotlib/sphinxext/mathmpl.py

```diff
@@ -118,9 +118,6 @@ def depart_latex_math_latex(self, node):
                  latex=(visit_latex_math_latex, depart_latex_math_latex))
     app.add_role('mathmpl', math_role)
     app.add_directive('mathmpl', MathDirective)
-    if sphinx.version_info < (1, 8):
```

Comment:
that's for 3rd party users, not for us.  this can still be removed (as :mathmpl: is a replacement), but needs an API changes note.

Reply:
Reverted. I assume this would need deprecation before removal; and I don't have the time right now how to do this properly.